collector/exchange.go

```diff
@@ -0,0 +1,679 @@
+// +build windows
+
+package collector
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const subsystem string = "exchange"
```

Comment:
This will "pollute" the entire package, which might lead to unexpected usages elsewhere. Could we move it into the func where it is used?

Reply:
Yes, definitely